src/main/resources/application.properties

```diff
@@ -8,3 +8,6 @@ spring.datasource.password=password
 spring.jpa.hibernate.show_sql=true
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.open-in-view=false
+
+jwt.expiration=3000000
+jwt.secret=superStrongSecretPassword%%%%%%%%%%%%
```

Comment:
It's better to store secrets of main application.properties in some environment/system. It's a bad prictice to push them to git. (for test application.properties you can leave any mock secrets in order for CI to pass)

Reply:
Yeah, I know. I would store it in application-prod.properties. But if I creat application-dev.prorerties now, and not push them to git, no one can build this app without it.